Jenkinsfile

```diff
@@ -0,0 +1,2 @@
+def pipe = new KafkaConnectPipeline()
```

Comment:
Are you using Jenkins? If not, this file can be removed.

Reply:
Ok I've removed the file.